src/controllers/authenticationController.ts

```diff
@@ -297,9 +327,19 @@ export default class AuthenticationController {
               )
             )
           ),
-          TE.mapLeft(() =>
-            O.some(ResponseErrorInternal("Error Activation Lollipop Key"))
-          )
+          TE.mapLeft(error => {
```

Comment:
Split this event in two. The first inside the lollipop service (only the catch error and the response decode errors), the second instead inside the pipe of `setLollipopAssertionRefForUser`

Reply:
done https://github.com/pagopa/io-backend/pull/996/commits/acab1e4c85b2ffeb63e2296d387f274629178eab